Makefile

```diff
@@ -1,11 +1,14 @@
-.PHONY: up down install ci stan cs cs-fix test migrate
+.PHONY: up down install ci stan cs cs-fix test migrate ssh
 
 up:
 	docker-compose up -d
 
 down:
 	docker-compose down
 
+ssh:
```

Comment:
I added the same command but with name `bash`

Reply:
I just missed it, I've removed it from this PR.